Bucardo/Installation.md

```diff
@@ -40,8 +40,11 @@ sudo make install
 
 Other Perl prerequisites that may or may not be installed already on your system include:
 
+- `CGI`
```

Comment:
This is only used by the bucardo-report, which is not a core part of Bucardo, and which I haven't used before. So it's not a prereq for Bucardo itself. Maybe just add ` (only needed for bucardo-report)`?